src/scripts/add-implementation.js

```diff
@@ -1,8 +1,7 @@
 import AppController from "../models/AppController";
 
-export default function addImplementation({ contractName, contractAlias, packageFileName = undefined }) {
-  if (contractName === undefined) throw new Error('Must provide a contract name')
-  if (!contractAlias) contractAlias = contractName
+export default function addImplementation({ contractName, contractAlias = contractName, packageFileName = undefined }) {
```

Comment:
Are we sure that this works fine in JS? It doesn't in Python, hence the question.

Reply:
yes it works, however I'm pushing a new PR to support adding multiple implementations that changes this script